tasks/functions/functions.js

```diff
@@ -20,38 +25,68 @@ function sum(a, b) {
  *    lastName: "Dou"
  * }
  */
-function getFullName(object) {
 
+/*const object={
+		firstName: "Jenna",
+		lastName: "Luchynyn"
+	
+	}*/
+function getFullName(object) {
+  return object.firstName + " " + object.lastName;
 }
 
+//console.log(getFullName(object));
+
 /**
  * write fuction that checks if number is odd
  * true if odd, false if even
  */
 function isOdd(n) {
-
+  if (n % 2 == 0) {
+    return false;
+  }
+  return true;
```

Comment:
1. Should be simplified: the result of the equation operator is already boolean.
2. Please use strict equation.